### DISCUSSION TITLE
Where can I find gpodder's settings?

### DISCUSSION THREAD

**Discussion by nunesgh** [thrillfall/nextcloud-gpodder#57] [Q&A] [answered]
Hi there,

Thank you for this app!

I have just installed it on my Nextcloud instance, but I could not find its settings page anywhere.

How are we supposed to manage gpodder from within Nextcloud? Or is there another way for doing so?

Thank you for your attention!

**Comment by JonOfUs** [comment, marked as answer]
Hi,
there are no settings for this app.

You can connect e.g. AntennaPod with gpoddersync as well as KDE Kasts and this way, your play times and subscriptions will be synchronized between both apps. The synchronization happens fully in the background of your Nextcloud and you can only see the stored data if you connect a podcast app to it.

**Reply by nunesgh** [reply]
Thank you for your answer, @JonOfUs!

But how to manage users and credentials? Is it safe to have no way of managing settings?

**Reply by JonOfUs** [reply]
The users and credential systems of Nextcloud is used. To connect with a podcast app, you have to provide your Nextcloud account credentials in the podcast app (or allow access to your account by this app).

I can't think of settings that one should be able to change.

**Reply by nunesgh** [reply]
Cool! Thank you for clarifying how the app works!

**Reply by thrillfall** [reply]
Thanks @JonOfUs for doing the user support...again ;-)

Should we add a note in the Readme that there are no settings to be set?

**Reply by JonOfUs** [reply]
Always happy to help if I can :)

Yeah, maybe a note about this being a backend api wouldn't be bad. Maybe something like "This app only acts as a synchronization endpoint and therefore has no UI or configuration options." in the description of the app (in Readme).
